src/services/tx/safeUpdateParams.ts

```diff
@@ -14,13 +44,17 @@ export const CHANGE_FALLBACK_HANDLER_ABI = 'function setFallbackHandler(address
  */
 export const createUpdateSafeTxs = (safe: SafeInfo, chain: ChainInfo): MetaTransactionData[] => {
   const latestMasterCopy = getGnosisSafeContractInstance(chain, LATEST_SAFE_VERSION)
-  const safeContractInstance = getGnosisSafeContractInstance(chain, safe.version)
+  const safeContractInterface = getContractInterface(chain, safe.version)
 
   // @ts-expect-error this was removed in 1.3.0 but we need to support it for older safe versions
-  const changeMasterCopyCallData = safeContractInstance.encode('changeMasterCopy', [latestMasterCopy.getAddress()])
+  const changeMasterCopyCallData = safeContractInterface.encodeFunctionData('changeMasterCopy', [
```

Comment:
It looks like `changeMasterCopy` is not found on the 1.0.0 contract even though the function exists.

<img width="999" alt="Screenshot 2022-09-08 at 10 20 36" src="https://user-images.githubusercontent.com/5880855/189072821-0dbcd759-f90f-4089-97b3-9ef7eb85cbc5.png">